package.json

```diff
@@ -120,7 +120,8 @@
     "stream": "0.0.2",
     "stream-browserify": "1.0.0",
     "url": "0.10.1",
-    "uuid": "8.3.2"
+    "uuid": "8.3.2",
+    "libportal-react-native": "git+https://github.com/cakesoft-vaibhav/libportal-react-native.git"
```

Comment:
Why is it not on Bithyve? Also ideally it should specify a commit hash not just the branch

Reply:
Like CKTap  "cktap-protocol-react-native": "git+https://github.com/bithyve/cktap-protocol-react-native.git#7be02d23550e88a5a0b5790138734d34c688ef5d",

Reply:
fixed